src/internal/symbol/observable.ts

```diff
@@ -1,2 +1,5 @@
 /** Symbol.observable or a string "@@observable". Used for interop */
-export const observable: string | symbol = (() => (typeof Symbol === 'function' && Symbol.observable) || '@@observable')();
+export const observable = (() => (typeof Symbol === 'function' && Symbol.observable) || '@@observable')();
+export const observableTypeRef = {
+  symbol: observable as Exclude<typeof observable, string>,
+};
```

Comment:
Yeah, this is weird - I don't know how to declare it in a better way that would preserve the `typeof Symbol.observable` for this. You could use this [TS playground](https://www.typescriptlang.org/play?ts=4.5.4#code/KYDwDg9gTgLgBAbwL4CgAmwDGAbAhlYOAc2wgCNdtEU44BLAOxmCgDNdNCBlATwFsyEbAGEIDAM4woAV0wxo1WrQK40Y7DzjlxALjjj+g7DTipUKTGMlxS1gLxxeAoQDptKAPQelAPQD8nt4A7nQwABYQ0vDiEHyEANaMaFqscLhwQVC4YGAscKzSDHJ0YlpQ+nzhcOHchq7a1Ty5xMAw4jYQkhZW0fxwDgAUAwCU-QB8cAMwTcAQqU5G-XYOAOQFRTAlDCtwAGS7jnXYbmTiowA+53ArAAI32iwAbrhk2MArwyMA3IG+Ad0SXp8YS4STAZIOAx8NLtACiIBw0gwAB5prk5vp+AAafRSRhEMYoAHWKEAdSyOTyDgQmL4eihMLg8MRKLRs1SUJxkig+ImqC8fyJoEgsDgOFB7QAgoo4ABtWwwAC6ehG40QZiF4Gg8HF4naACEZbKocrJqM7BNkChzMLtWK8Hq4MIjVCQWC0KbVRb1dbNSKdQ72gARF38cnZXJQFwmlXmy0alCMZhsDiEUmhMIASSYLAgYGRABVLSZZQt6qdPXG4AWfjataK2XBM+JMzBSdBEgwiP04CHQMwGGh2unwtnk3nkbgGDwJn44ABGOB6AAMvyU67g-iAA) to experiment with this more

Reply:
Actually, I've managed to simplify this by casting the IFFE's result to any and assigning to `: typeof Symbol.observable`. For some reason `as typeof Symbol.observable` doesn't work and this was what I was previously attempting to do.